phpBB/adm/style/overall_footer.html

```diff
@@ -30,11 +30,11 @@
 		</div>
 
 		<div id="phpbb_alert" class="phpbb_alert" data-l-err="{L_ERROR}" data-l-timeout-processing-req="{L_TIMEOUT_PROCESSING_REQ}">
-			<a href="#" class="alert_close"></a>
+			<a href="#" class="alert_close">{{ Icon('font', 'circle-xmark', '', true) }}</a>
```

Comment:
Somehow this popup looks different in the ACP when compared to prosilver but should basically look the same. Can you check what's wrong here?